Mathlib/Algebra/Module/CharacterModule.lean

```diff
@@ -126,4 +126,88 @@ Linear maps into a character module are exactly characters of tensor product.
 
 end module
 
+
+/--
+`ℤ⋆`, the character module of `ℤ` in rational circle
+-/
+protected abbrev int : Type := CharacterModule ℤ
+
+/-- Given `n : ℕ`, the map `m ↦ m / n`. -/
+protected abbrev int.divByNat (n : ℕ) : CharacterModule.int  :=
+  LinearMap.toSpanSingleton ℤ _ (QuotientAddGroup.mk (n : ℚ)⁻¹) |>.toAddMonoidHom
```

Comment:
You may consider changing this to a Z-linear map if you're not using it elsewhere, because when defining `ofSpanSingleton` you're converting it back to an LinearMap.

Reply:
What do you think about this?

Reply:
Thanks. I was trapped in marking today. Sorry for the late reply